src/components/section-list.component.js

```diff
@@ -69,9 +72,7 @@ export const SectionList = ({
   let listDisplay;
 
   if (loading) {
-    listDisplay = (
-      <ActivityIndicator animating={loading} style={styles.loadingIcon} />
-    );
+    listDisplay = <LoadingIcon animating={loading} />;
   } else if (noItems) {
     listDisplay = (
       <ListItem
```

Comment:
Can now be styled like this:

```js
const SectionListItem = styled(ListItem).attrs({
  titleStyle: {
     color: colors.black,
      ...fonts.fontPrimary,
  }
})``;
```

Reply:
Thanks @machour I'll take a look, I was on vacation.

Reply:
@machour Updated code accordingly, check it out 👍